www/DeviceAccounts.js

```diff
@@ -8,8 +8,12 @@ DeviceAccounts.prototype.get = function (onSuccess, onFail) {
 DeviceAccounts.prototype.getByType = function (type, onSuccess, onFail) {
     cordova.exec(onSuccess, onFail, 'DeviceAccounts', 'getDeviceAccountsByType', [type]);
 };
-DeviceAccounts.prototype.getPermissions = function (type, onSuccess, onFail) {
-    cordova.exec(onSuccess, onFail, 'DeviceAccounts', 'getPermissions', [type]);
+DeviceAccounts.prototype.getPermissions = function (type,onSuccess, onFail) {
+    let accountType = 'com.google';
+    if (typeof type === 'string') {
+        accountType =  type;
```

Comment:
Extra space between the `=` and `type`, should be `accountType = type;`

Reply:
resolved